### ISSUE TITLE
Opti.to_function() print-options are ignored

### ISSUE THREAD

**Issue by s6nke** [casadi/casadi#3838]
When construction an OCP via the Optistack and provide options to suppress printout there is no print when calling `opti.solve().` 
But when constructing the wrapper function `opti.to_function()`, printout is back. Am I missing something here?

```
opti_obj = Opti();
...
% construcion of OCP
...
opts = struct();
opts.error_on_fail = true;
opts.print_time = false;
opts.ipopt.print_level = 0;
opti_obj.solver('ipopt', opts);

opti_func = opti_obj.to_function('opti_func',...
        [parameters,decisionVariables], ...
        decisionVariables); 
%
opti_obj.solve(...) % -> no print
opti_func(...) % -> print is back
```

**Comment by s6nke** [comment]
Thanks a lot for you quick answer. Unfortunately, I am quite sure that I am using the latest prebuild casadi-3.6.6-linux64-matlab2018b. At least, thats the only casadi on my matlab-path.

**Comment by jgillis** [comment]
Ah, my bad, it was fixed on main, not yet in a release, see #3805
